src/main/java/cart/dto/request/OrderItemRequest.java

```diff
@@ -0,0 +1,28 @@
+package cart.dto.request;
+
+import java.beans.ConstructorProperties;
+import javax.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Range;
+
+public class OrderItemRequest {
+
+    @NotNull
+    private Long id;
+    @NotNull
+    @Range(min = 1)
+    private Integer quantity;
```

Comment:
```suggestion
    @NotNull
    private Long id;
    
    @NotNull
    @Range(min = 1)
    private Integer quantity;
```
애노테이션이 있으면 이렇게 개행해주는 것이 가독성에 더 도움될 것 같아요

Reply:
동의합니다!!👍 수정하겠습니다